packages/modules/devices/kaco/kaco_tx/device.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import logging
+from typing import Iterable
+
+from modules.common import modbus
+from modules.common.abstract_device import DeviceDescriptor
+from modules.common.configurable_device import ComponentFactoryByType, ConfigurableDevice, MultiComponentUpdater
+from modules.devices.kaco.kaco_tx.inverter import KacoInverter
+from modules.devices.kaco.kaco_tx.config import (Kaco, KacoInverterSetup)
+
+log = logging.getLogger(__name__)
+
+default_unit_id = 85
+synergy_unit_identifier = 160
+reconnect_delay = 1.2
+
+
+def create_device(device_config: Kaco):
+    client = None
+
+    def create_inverter_component(component_config: KacoInverterSetup):
+        nonlocal client
+        return KacoInverter(component_config, client=client, device_id=device_config.id)
```

Comment:
```suggestion
        return KacoInverter(component_config, client=client)
```
Auf die Device-ID wird in der Komponente nicht zugegriffen.